core/status.go

```diff
@@ -23,20 +23,23 @@ import (
 	"fmt"
 	"net/http"
 	"strings"
+	"time"
 
 	"github.com/labstack/echo/v4"
 )
 
 const moduleName = "Status"
 
 type status struct {
-	system *System
+	system    *System
+	startTime time.Time
 }
 
 //NewStatusEngine creates a new Engine for viewing all engines
 func NewStatusEngine(system *System) Engine {
 	return &status{
-		system: system,
+		system:    system,
+		startTime: time.Now(),
```

Comment:
I just read something about Monotonic clocks. It seems that the golang Time does both correct! 😱

Reply:
Nice, didnt realize that's a thing. Question is however, what 'correct' is; if your goal is to display a time relative to the absolute "world time" (e.g. signing time for a document), I'd use the wall clock. For this use case (server up for XXX seconds) I'd use the monotonic clock.